Yank/analyze.py

```diff
@@ -1055,6 +1099,19 @@ def analyze_phase(self, cutoff=0.05):
         data['DeltaF_standard_state_correction'] = self.get_standard_state_correction()
         return data
 
+class ReplicaExchangeAnalyzer(MultiStateSamplerAnalyzer):
```

Comment:
I created this subclass in case there needed to be a one-to-one mapping between the analyzer for a sampler class and the sampler class itself. So far, I haven't found the need to move anything into it, but I imagine there will be replica-exchange specific analysis features we may want there in the future.